### PR TITLE
Get name Element text

### PR DESCRIPTION
**Pull request by cannedmoose**

### CODE REVIEW

**Comment by AshleyYakeley** [comment]
Thanks. This change needs to go elsewhere too. Can you look at #13?

File: tool/nixvirt.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ def _descriptionXMLText(self,lvobj):
         # VIR_NETWORK_XML_INACTIVE
         return lvobj.XMLDesc(flags=1)
     def _getDependents(self,obj):
-        networknames = [str(name) for name in obj.descriptionXMLETree().xpath("/network/name")]
+        networknames = [str(name.text) for name in obj.descriptionXMLETree().xpath("/network/name")]
```
Comment (AshleyYakeley) [inline tool/nixvirt.py:124]: This doesn't need `str()`, does it?

**Comment by AshleyYakeley** [comment]
It's times like this I really miss Haskell...

File: tool/nixvirt.py (hunk 1 above)
Comment (AshleyYakeley) [reply tool/nixvirt.py:124]: I'm OK with using `str()` for the other ones, because I'd prefer to convert "smart strings" to strings before comparison.

Comment (cannedmoose) [reply tool/nixvirt.py:124]: Actually you're right - no need to do the conversion, from what I can see [in the docs](https://lxml.de/api/lxml.etree._Element-class.html) this is just a regular string!

**Comment by cannedmoose** [comment]
> It's times like this I really miss Haskell...

Haha probably too much of a yak shave to write some libvirt bindings for haskell

File: tool/nixvirt.py (hunk 1 above)
Comment (cannedmoose) [reply tool/nixvirt.py]: I've made the change but haven't tested on my machine yet - will do that within the next hour just to double check the docs haven't lied...

**Comment by AshleyYakeley** [comment]
Awesome, thanks.

> Haha probably too much of a yak shave to write some libvirt bindings for haskell

Yeah someone [started](https://hackage.haskell.org/package/libvirt-hs) but it looks incomplete and broken.

**Comment by cannedmoose** [comment]
Confirmed working for me :+1:

**Pull request merged**